src/models/Config.d.ts

```diff
@@ -39,6 +40,7 @@ export interface configType {
   cors_origin: string
   cors_headers: string
   cors_methods: string
+  connection_string: string
```

Comment:
should this be optional?

Reply:
hmm, maybe -- i supposed we can eliminate the "use_db" option and just check if connection string is null or empty? That might be better option? what do ya think?

Reply:
personally, i like the separate bool to disable the db. then we could have a default localhost string in the config. might be a little less intuitive if we go with just the connection string.